bigframes/core/indexes/base.py

```diff
@@ -424,6 +433,8 @@ def dropna(self, how: typing.Literal["all", "any"] = "any") -> Index:
         return Index(result)
 
     def drop_duplicates(self, *, keep: str = "first") -> Index:
+        if keep is not False:
+            enforce_ordered(self, "duplicated")
```

Comment:
Is this a typo? I assume it should reflect the public API name used, right?

```suggestion
            enforce_ordered(self, "drop_duplicates")
```

Reply:
fixed